interfaces/Portalicious/src/app/models/payment.model.ts

```diff
@@ -0,0 +1,7 @@
+// TODO fix all strict problems in 121 service to use the correct types
```

Comment:
```suggestion
// TODO: fix all strict problems in 121 service to use the correct types
```

I'm not sure why but this is common practice